async_packager/setup.cfg

```diff
@@ -0,0 +1,39 @@
+[metadata]
+name = cumulus-packager
+version = 0.1.0
```

Comment:
I would call this 2.0

Reply:
Cumulus-packager is separate from cumulus-geproc but with similarities.  I'd prefer having one package for both containers; less to maintain.